tutorials/graphs/timeonaxis.C

```diff
@@ -81,25 +81,23 @@ TCanvas *timeonaxis()
    gt->GetXaxis()->SetTimeDisplay(1);
    gPad->Modified();
 
-//### Build a second simple graph for a very long time interval 
-//        Time interval : a few years
+//### Build a second simple graph for a very long time interval
+//        Time interval: a few years
 
-   float x2[10], t2[10];
-   for (i=0;i<10;i++) {
-      x2[i] = gRandom->Gaus(500,100)*i;
-      t2[i] = i*365*86400;
-   }
-   TGraph *gt2 = new TGraph(10,t2,x2);
+   auto gt2 = new TGraph();
+   int nd;
```

Comment:
And in case of TGraph one can put exact date like:
```
   TDatime dateBegin(2000,1,1,0,0,0);
   for (i=0;i<10;i++) {
       TDatime datePnt(2000 + i,1,1,0,0,0);
      gr->AddPoint(datePnt.Convert() - dateBegin.Convert(), 100 + gRandom->Gaus(500,100)*i);
   }
```

Reply:
oops ... a left over from a try I did